app/src/store/cart.ts

```diff
@@ -52,10 +31,50 @@ const SWAP_PATHS = {
-  '0x6B175474E89094C44Da98b954EedeAC495271d0F': '0x6b175474e89094c44da98b954eedeac495271d0f',
+  '0x6B175474E89094C44Da98b954EedeAC495271d0F': '0x6B175474E89094C44Da98b954EedeAC495271d0F'.toLowerCase(),
+};
+const SWAP_PATHS = {
+  [SupportedChainId.MAINNET]: MAINNET_SWAP_PATHS,
+  [SupportedChainId.HARDHAT]: MAINNET_SWAP_PATHS,
+  [SupportedChainId.RINKEBY]: {
+    // ETH to DAI through the 0.3% pool
+    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE': '0xc778417e063141139fce010982780140aa0cd5ab000bb85592ec0cfb4dbc12d3ab100b257153436a1f0fea', // prettier-ignore
+    // DAI "swap path" is just its token address for our router
+    '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa': '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa'.toLowerCase(),
+  },
+  [SupportedChainId.OPTIMISM]: {
+    // ETH to DAI through the 0.3% pool
+    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE': '0x4200000000000000000000000000000000000006000bb8da10009cbd5d07dd0cecc66161fc93d7c9000da1', // prettier-ignore
+    // USDC to DAI through the 0.3% pool
+    '0x7F5c764cBc14f9669B88837ca1490cCa17c31607': '0x7f5c764cbc14f9669b88837ca1490cca17c31607000bb8da10009cbd5d07dd0cecc66161fc93d7c9000da1', // prettier-ignore
+    // DAI "swap path" is just its token address for our router
+    '0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1': '0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1'.toLowerCase(),
+  },
+  [SupportedChainId.OPTIMISTIC_KOVAN]: {
+    // ETH to DAI through the 0.3% pool
+    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE': '0x4200000000000000000000000000000000000006000bb8da10009cbd5d07dd0cecc66161fc93d7c9000da1', // prettier-ignore
+    // USDC to DAI through the 0.3% pool
+    '0x7F5c764cBc14f9669B88837ca1490cCa17c31607': '0x7f5c764cbc14f9669b88837ca1490cca17c31607000bb8da10009cbd5d07dd0cecc66161fc93d7c9000da1', // prettier-ignore
+    // DAI "swap path" is just its token address for our router
+    '0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1': '0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1'.toLowerCase(),
+  },
+  // *** Arbitrum paths are currently not handled as there is no bridged DAI ***
+  // We use the mainnet paths here just to avoid lint errors
+  [SupportedChainId.ARBITRUM_ONE]: {
+    // ETH to USDC through the 0.3% pool
+    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE': '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1000bb8ff970a61a04b1ca14834a43f5de4533ebddb5cc8', // prettier-ignore
+    // USDC "swap path" is just its token address for our router (no DAI on Arbitrum yet, so using USDC instead)
+    '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8': '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8'.toLowerCase(),
+  },
+  [SupportedChainId.ARBITRUM_RINKEBY]: {
+    // Seems Uniswap V3 has no liquidity here (this swap path is not correct)
+    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE': '0x82aF49447D8a07e3bd95BD0d56f35241523fBab1000bb86b175474e89094c44da98b954eedeac495271d0f', // prettier-ignore
+    // TODO choose donation token since there's no DAI -- USDC?
+  },
```

Comment:
^ would make sense to push this to it's file after MVP launch

Reply:
After MVP we'll replace this with a better solution: #140